python/repositories.bzl

```diff
@@ -154,6 +168,7 @@ py_runtime(
     files = [":files"],
     interpreter = "{python_path}",
     python_version = "PY3",
+    {stub_shebang_assignment}
```

Comment:
I think this functionality could be behind a default `True` configuration knob. I'm not confident it will be the right move in all scenarios. 

When doing `bazel run` (or `test`) it will work, but if the `py_binary` is moved in any way, for example to layer it into a container, then the absolute path will not be valid. 

With 'in-build' Python toolchains, I'm pretty sure the toolchain interpreter is moved with the rest of the application files. So it can be found at a relative path to the stub_script, for example `foo.runfiles/python39_x86_64-apple-darwin/bin/python3`.